crates/edr_provider/src/data.rs

```diff
@@ -2759,6 +2759,7 @@ mod tests {
         Ok(())
     }
 
+    #[cfg(feature = "test-remote")]
```

Comment:
Right, I see your point but this would break local symmetry of some tests, e.g. `test_local_account_balance` and `test_local_account_balance_forked`.

As a middle ground, I've separated the tests that explicitly call `get_alchemy_url` and bundled them together but for simpler tests that just use `new_local` and `new_forked` and are close to each other I left alone.

I think this strikes a nice balance; let me know what you think.

Reply:
Sounds good to me